src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -16,12 +16,12 @@
 
 namespace Coverlet.Core.Symbols
 {
-    internal static class CecilSymbolHelper
+    internal class CecilSymbolHelper : ICecilSymbolHelper
     {
         private const int StepOverLineCode = 0xFEEFEE;
         private static ConcurrentDictionary<string, int[]> CompilerGeneratedBranchesToExclude = null;
```

Comment:
Remove `static` semantic is the same with singleton.

Reply:
Done